src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -263,9 +282,11 @@ public void validateArguments() {
 
         // throw if args were specified for a filter that was also disabled
         disableFilters.forEach(s -> {
-            if (requiredPredecessors.contains(s)) {
+            if (requiredPredecessors.contains(s) && !toolDefaultReadFilters.containsKey(s)) {
                 throw new CommandLineException(
                         String.format("Values were supplied for (%s) that is also disabled", s));
```

Comment:
Right, I update these code paths a bit and updated the tests for this so that there are test that execute all the code paths. The test you mentioned is still disabled, but I added comments about why we need to work around the Barclay behavior.